extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -96,6 +96,8 @@ export class AmpAdXOriginIframeHandler {
           this.element_.creativeId = info.data.id;
         });
 
+    // TODO(keithwrightbos) - move to within not A4A else block to ensure this
```

Comment:
this API should be available to both fast fetch and delayed fetch, right? why is the TODO?